build.gradle.kts

```
@@ -1,12 +1,15 @@
 plugins {
-	id("dev.yumi.gradle.licenser").version("1.1.+")
-	id("com.gradle.plugin-publish").version("1.2.0")
-	id("maven-publish")
-	id("signing")
+	id("dev.yumi.gradle.licenser") version "1.1.+"
+	id("com.gradle.plugin-publish") version "1.2.0"
+
+	kotlin("jvm") version "1.9.24"
+
+	`maven-publish`
+	signing
 }
 
 group = "dev.yumi"
-version = "1.1.2"
+version = "1.1.3"
```

```suggestion
version = "1.2.0"
```

Given that's new features I'd increment the minor version instead of the patch.